addrs/output_value.go

```diff
@@ -13,6 +13,7 @@ import (
 // OutputValue is not Referenceable, while ModuleCallOutput is.
 type OutputValue struct {
 	Name string
+	referenceable
```

Comment:
>  // OutputValue is not Referenceable, while ModuleCallOutput is.

The comment right above this would be incorrect from this PR, and as well, I wonder about how this relates to this code's intention.

Reply:
ah wonderful catch, thanks! The whole point is that it is now referenceable so MILDLY IMPORTANT that the comments match